examples/user_guide/Dependencies_and_Watchers.ipynb

```diff
@@ -0,0 +1,820 @@
+   "cell_type": "markdown",
+   "id": "deacc3cd",
+   "metadata": {},
+   "source": [
+    "If you trigger `a`, the usual series of chained events will be triggered, including changing `b`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9667d7ea",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "p.param.trigger('a')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0b605755",
+   "metadata": {},
+   "source": [
+    "### `Event` Parameter\n",
+    "\n",
+    "An Event Parameter is a special Parameter type whose value is intimately linked to the triggering of events for Watchers to consume. Event has a Boolean value, which when set to `True` triggers the associated watchers (as any Parameter does) but then is automatically set back to `False`. \n",
+    "\n",
+    "Conversely, if events are triggered directly on a `param.Event` via `.trigger`, the value is transiently set to True (so that it's clear which of many parameters being watched may have changed), then restored to False when the triggering completes. An Event parameter is thus like a momentary switch or pushbutton with a transient True value that normally serves only to launch some other action (e.g. via a `param.depends` decorator or a watcher), rather than encapsulating the action itself as `param.Action` does. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "70a2937e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class Q(param.Parameterized):\n",
+    "    e = param.Event()\n",
+    "    \n",
+    "    @param.depends('e', watch=True)\n",
+    "    def callback(self):\n",
+    "        print(f'e=={self.e}')\n",
+    "        \n",
+    "q = Q()\n",
+    "q.e=True"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "62919cfe",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "q.e"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d4d11da7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "q.param.trigger('e')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "57ac5634",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "q.e"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "822c7365",
+   "metadata": {},
+   "source": [
+    "### Async executor\n",
+    "\n",
+    "Param's events and callbacks described above are all synchronous, happening in a clearly defined order where the processing of each function blocks all other processing until it is completed. Watchers can also be used with the Python [asyncio](https://docs.python.org/3/library/asyncio.html) library to operate asynchronously. To do this, you can define `param.parameterized.async_executor` with an asynchronous executor that allows using coroutines and other asynchronous functions as `.param.watch` callbacks. These callbacks will be scheduled on the event loop from Tornado or asyncio. \n",
```

Comment:
You don't need to mention `asyncio`. The important thing to mention is support for `async/await` syntax, which is part of the core language. Whether you're using an `asyncio`, [`curio`](https://github.com/dabeaz/curio),  [`trio`](https://github.com/python-trio/trio) or whatever other async event loop implementations exist doesn't much matter.